remote/remote_test.go

```diff
@@ -20,6 +26,15 @@ import (
 )
 
 var dockerRegistry, readonlyDockerRegistry *h.DockerRegistry
+var count int
```

Comment:
I can see why you did it this way, but is there any way to avoid making this a global variable?

Reply:
Any suggestion will be great!, let me try to find some other way to do it

Reply:
Maybe when you make [these changes](https://github.com/buildpacks/imgutil/pull/111#discussion_r627863324), it'll be easier to move this global variable.

Reply:
I am not sure if this would work, but perhaps the server could hold on to a map of endpoints -> count of times accessed. Then you could have a separate endpoint to retrieve the map? IDK

Reply:
I created a new data structure wrapping the http server and saving the number of methods calls internally, in that way I managed to remove the global variable